user-guide/Advanced_Functionality/DataMiner_Systems/Adding_a_DMA/Adding_a_DMA_to_a_DMS_running_STaaS.md

```diff
@@ -8,55 +8,31 @@ To add a DataMiner Agent to a DMS running STaaS:
 
 1. Make sure that the DMA you are adding is a clean DMA, meaning that it is a newly installed DMA or a DMA that has been [fully reset](xref:Factory_reset_tool).
 
-   > [!IMPORTANT]
-   > If the CloudGateway DxM is installed on the DMA you are adding, the data folder `C:\ProgramData\Skyline Communications\DataMiner CloudGateway\Data` must be empty.
+1. Install the [DataMiner Cloud Pack](xref:CloudPackages) on the new DataMiner agent.
 
-1. Stop the DMA that will be added.
+1. Stop the DMA and CloudGateway on the server that will be added.
 
-1. On the DMA that will be added, stop CloudGateway if it is installed.
+1. From a DMA of the cluster where a CloudGateway is running. Copy the `C:\ProgramData\Skyline Communications\DataMiner CloudGateway\Data\CcaGatewayPersisted` folder and place it in the same location on the new DMA.
 
-1. On the DMA that will be added, stop both the NAS and NATS services.
+1. Remove the DmsAccessToken.data and DmsRefreshToken.data files from the folder you just copied to the new agent.
```

Comment:
Should the DmsAccessTokenExpiration.data and DmsRefreshTokenExpiration.data files be removed as well?

Reply:
Tested it without these files and all works fine

Reply:
Added the files